plasmapy/physics/tests/test_quantum.py

```diff
@@ -48,3 +60,37 @@ def test_deBroglie_wavelength():
 
     with pytest.raises(ValueError):
         deBroglie_wavelength(8*u.m/u.s, 'sddsf')
+
+# defining some plasma parameters for tests
+T_e = 1 * u.eV
+n_e = 1e23 * u.cm**-3
+# should probably change this to use unittest module
+# add tests for numpy arrays as inputs
+# add tests for different astropy units (random fuzzing method?)
+def test_thermal_deBroglie_wavelength():
+    r"""Test the thermal_deBroglie_wavelength function in quantum.py."""
+    lambda_dbTh = thermal_deBroglie_wavelength(T_e)
+    # test a simple case for expected value
+    assert np.isclose(lambda_dbTh.value,
+                      6.919367518364532e-10)
```

Comment:
Related to #150: doing it like this:

    assert np.isclose(lambda_dbTh.value, 6.919367518364532e-10), "Thermal deBroglie wavelength is off expected result for 1eV"

... or with some other short explanation could be much more readable. The string would be thrown on assertion failure in PyTest.